ginga/misc/Datasrc.py

```diff
@@ -65,8 +65,8 @@ def pop(self, *args):
         if len(args) == 0:
             return self.remove(self.history[0])
 
-        assert len(args) == 1, \
-               ValueError("Too many parameters to pop()")
```

Comment:
Note: I see several cases of doing `assert` only to raise a different exception. Perhaps it is cleaner to use `if` instead, as I demonstrated here. If you want to pursue this for all such cases in this package, it is a separate PR for another day.